meetings/2024-11-22.md

```diff
@@ -0,0 +1,49 @@
+<img src="https://tc54.org/images/ecma.svg" align="right" height="70" alt="Ecma logo" /> <!-- markdownlint-disable-line MD041 -->
+
+# Agenda for the seventh meeting of Ecma TC54-TG2 on 2024-11-22
+
+- **Host**: Remote
+- **Dates and times**:
+    - 14:00 to 15:00 UTC
+      - 16:00 to 17:00 CEST (Europe/Brussels)
+      - 10:00 to 11:00 EDT (America/New_York)
+      - 07:00 to 08:00 PDT (America/Los Angeles)
+      - 23:00 to 00:00 JST (Tokyo, Japan)
+- **Attendee information**:
+  - https://meet.google.com/ayu-mpvf-ycw
+  - [Meeting invite](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NXVqbW9mMDMzaDVuZmlsMDdsZWZsMm9kNXVfMjAyNDEwMjVUMTQwMDAwWiBjX2Q4YjE1NDIwZGZmMTdiNzk1OWUyOWE1MWFlMzI0MDk1MWNiZTM4ZGIxZGFlNDU5NzJhODVjOWE3MTEyMDQyMDVAZw&tmsrc=c_d8b15420dff17b7959e29a51ae3240951cbe38db1dae45972a85c9a711204205%40group.calendar.google.com&scp=ALL)
+
+## Agenda items
+- Opening, welcome, and roll call (Chair)
+- Review last call minutes: https://github.com/Ecma-TC54/tg2/blob/main/meetings/2024-11-08.md
+- Update on list of issues and PRs for Ecma spec (John):
+https://docs.google.com/spreadsheets/d/1H2QAcADLaMNgcR5BMK7bQxzH5D3X-SdO/
+- Steve: What is needed to put into the Ecma spec?
+
+## Attendees
+- Philippe Ombredanne, creator of PURL, Lead maintainer of AboutCode, TC54-TG2 convener
+- Steve Springett, OWASP Foundation / ServiceNow
+- Jannis Hermanns, Apple
+- Matt Rutkowski, IBM
+- Dustin Ingram
```

Comment:
```suggestion
- Dustin Ingram, Google 
```

Reply:
Hi @di and thanks for catching that!  Will update momentarily.  👍

Reply:
Updated -- thanks again @di .